src/quo2/components/avatars/icon_avatar.cljs

```diff
@@ -22,6 +22,8 @@
      {:style {:width            component-size
               :height           component-size
               :border-radius    component-size
+              :border-width     (when border? 1)
+              :border-color     (colors/theme-colors colors/neutral-20 colors/neutral-80)
```

Comment:
would it be possible that you update this component to use new theming mechanism while making these changes here?
i.e so it's using `(colors/theme-colors colors/neutral-20 colors/neutral-80 theme)`

Reply:
@J-Son89 done